model/ids.go

```diff
@@ -156,6 +156,11 @@ func (t *TraceID) ToOTELTraceID() pcommon.TraceID {
 	return traceID
 }
 
+func TraceIDFromOTEL(traceID pcommon.TraceID) TraceID {
+	tid, _ := TraceIDFromBytes(traceID[:])
```

Comment:
not exactly what I meant. I would move the 16-byte logic from `TraceIDFromBytes()` into this new function, and invoke this new function from `TraceIDFromBytes()`

Reply:
what about something like this? or do we want this to be more generic to just accept any 16 byte array?
```go
func TraceIDFromOTEL(traceID pcommon.TraceID) TraceID {
	return TraceID{
		High: binary.BigEndian.Uint64(traceID[:traceIDShortBytesLen]),
		Low:  binary.BigEndian.Uint64(traceID[traceIDShortBytesLen:]),
	}
}
```